spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/CassandraRow.scala

```diff
@@ -72,12 +72,66 @@ import com.datastax.driver.core.Row
   *   - java.sql.Date
   *   - org.joda.time.DateTime
   */
-final class CassandraRow(val columnNames: IndexedSeq[String], val columnValues: IndexedSeq[AnyRef])
+final class CassandraRow(val metaData: CassandraRowMetaData, val columnValues: IndexedSeq[AnyRef])
```

Comment:
run following code to get all Row Serialised:
cala> sc.cassandraTable("ks", "tab").repartition(2).count
res0: Long = 20000000

jvisualvm memory sampler show that number of meta store object is small:
com.datastax.spark.connector.CassandraRowMetaData	0.03575835062886026	129,600 B (0%)	3,240 (0%)
com.datastax.spark.connector.CassandraRow	1.0719691326853993	3,885,168 B (1.1%)	161,882 (1.9%)